apps/client/src/hooks/useLocaleChange.ts

```diff
@@ -23,5 +26,5 @@ export const useLocaleChange = () => {
     };
   }, []);
 
-  return locale;
+  return { locale, isRTL: dir === 'rtl' };
```

Comment:
because it's either he-IL or en-US, you don't need to maintain another state, and just pass:
```suggestion
  return { locale, isRTL: locale === 'he-IL' };
```